gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/metrics/KafkaReportingFormats.java

```diff
@@ -68,6 +70,10 @@
     switch (this) {
       case AVRO:
         KafkaAvroEventReporter.Builder<?> builder = KafkaAvroEventReporter.Factory.forContext(context);
+        if (properties.getProperty(PusherUtils.KAFKA_PUSHER_CLASS_NAME_KEY,
+                PusherUtils.DEFAULT_KAFKA_PUSHER_CLASS_NAME).equals(PusherUtils.KAFKA_KEY_VALUE_PUSHER_CLASS_NAME)) {
+          builder = new KafkaAvroEventKeyValueReporter.BuilderImpl(context, properties);
```

Comment:
Why not KafkaAvroEventKeyValueReporter.Factory.forContext() to be consistent with KafkaAvroEventReporter?